--- libtommath-0.42.0/makefile.shared~	2013-02-13 08:13:53.051234142 +0100
+++ libtommath-0.42.0/makefile.shared	2013-02-13 08:16:07.423726946 +0100
@@ -91,7 +91,7 @@ bn_mp_to_signed_bin_n.o bn_mp_to_unsigne
 objs: $(OBJECTS)
 
 $(LIBNAME):  $(OBJECTS)
-	libtool --mode=link gcc *.lo -o $(LIBNAME) -rpath $(LIBPATH) -version-info $(VERSION)
+	libtool --tag=CC --mode=link gcc *.lo -o $(LIBNAME) -rpath $(LIBPATH) -version-info $(VERSION)
 
 install: $(LIBNAME)
 	install -d -g $(GROUP) -o $(USER) $(DESTDIR)$(LIBPATH)
